crates/server/src/admin/service.rs

```diff
@@ -99,6 +99,16 @@ pub(crate) fn setup(
             "/add-client-key",
             post(handlers::add_client_key::add_client_key_handler),
         )
+        .route(
+            "/install-dev-application",
```

Comment:
Why do you need all these endpoints?

Also, keep them organized under
/dev/ something restful